llumnix/backends/vllm/llm_engine.py

```diff
@@ -110,6 +110,7 @@ def _process_model_outputs(
                         new_scheduled_seq_groups.append(scheduled_seq_group)
                         new_seq_group_metadata_list.append(seq_group_meta)
                         new_output.append(seq_group_output)
+                        self.scheduler.request_info[seq_group.request_id].completed_step += 1
```

Comment:
I remember we said completed steps can be translated to the number of generated tokens?

Reply:
The "completed_step" represents the number of steps executed on the backend instance, rather than the total number of steps for the request. 
The term "completed_step" may be not appropriate. I'll use "instance_completed_steps" instead.

Reply:
> I remember we said completed steps can be translated to the number of generated tokens?

The "completed_step" represents the number of steps executed on the backend instance, rather than the total number of steps for the request.
The term "completed_step" may be not appropriate. I'll use "instance_completed_steps" instead.